<html>

<head>
<meta HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=windows-1252">
<meta NAME="GENERATOR" CONTENT="Microsoft FrontPage 3.0">
<title>Braz J Med Biol Res - MHC-restricted antigen presentation and recognition:
constraints on gene, recombinant and peptide vaccines in humans</title>
</head>

<body LINK="#0000ff" VLINK="#800080" BGCOLOR="#ffffff">
<b><font SIZE="4" COLOR="#000080">

<p align="center">Brazilian Journal of Medical and Biological Research</font><br>
<font SIZE="2" COLOR="#000080">Print ISSN 0100-879X</font></b><br>
</p>

<table BORDER="0" WIDTH="100%">
  <tr>
    <td WIDTH="5%"></td>
    <td WIDTH="90%"><b>Braz J Med Biol Res, February 1999, Volume 32(2) 199-205</b><p><a
    NAME="Home"></a><font SIZE="5"><b>MHC-restricted antigen presentation and recognition:
    constraints on gene, recombinant and peptide vaccines in humans</b></font></p>
    <p>E. Cunha-Neto</p>
    <p>Laboratório de Imunologia de Transplantes, Instituto do Coração, Hospital das
    Clínicas, Faculdade de Medicina, Universidade de São Paulo, São Paulo, SP, Brasil</p>
    <a HREF="#Abstract"><p><img SRC="down.gif" ALIGN="BOTTOM" BORDER="0" VSPACE="0" HSPACE="0"
    width="12" height="12"></a> <a HREF="#Abstract">Abstract</a><br>
    <a HREF="#Text"><img SRC="down.gif" ALIGN="BOTTOM" BORDER="0" VSPACE="0" HSPACE="0"
    width="12" height="12"></a> <a HREF="#Text">Text</a><br>
    <a HREF="#References"><img SRC="down.gif" ALIGN="BOTTOM" BORDER="0" VSPACE="0" HSPACE="0"
    width="12" height="12"></a> <a HREF="#References">References</a><br>
    <a HREF="#Correspondence"><img SRC="down.gif" ALIGN="BOTTOM" BORDER="0" VSPACE="0"
    HSPACE="0" width="12" height="12"></a> <a HREF="#Correspondence">Correspondence and
    Footnotes</a></p>
    <hr ALIGN="LEFT" WIDTH="100%" SIZE="2">
    <p><a NAME="Abstract"></a><font SIZE="4"><b><font COLOR="#00007F">Abstract </font><a
    HREF="#Home"><img SRC="back.gif" ALIGN="BOTTOM" BORDER="0" VSPACE="0" HSPACE="0" LOOP="0"
    width="12" height="12"></a></b></font></p>
    <p>The target of any immunization is to activate and expand lymphocyte clones with the
    desired recognition specificity and the necessary effector functions. In gene, recombinant
    and peptide vaccines, the immunogen is a single protein or a small assembly of epitopes
    from antigenic proteins. Since most immune responses against protein and peptide antigens
    are T-cell dependent, the molecular target of such vaccines is to generate at least 50-100
    complexes between MHC molecule and the antigenic peptide per antigen-presenting cell,
    sensitizing a T cell population of appropriate clonal size and effector characteristics.
    Thus, the immunobiology of antigen recognition by T cells must be taken into account when
    designing new generation peptide- or gene-based vaccines. Since T cell recognition is
    MHC-restricted, and given the wide polymorphism of the different MHC molecules, distinct
    epitopes may be recognized by different individuals in the population. Therefore, the
    issue of whether immunization will be effective in inducing a protective immune response,
    covering the entire target population, becomes an important question. Many pathogens have
    evolved molecular mechanisms to escape recognition by the immune system by variation of
    antigenic protein sequences. In this short review, we will discuss the several concepts
    related to selection of amino acid sequences to be included in DNA and peptide vaccines.</p>
    <p><b>Key words:</b> vaccines, MHC, antigen recognition, antigen processing, T cells,
    molecular evolution</p>
    <hr ALIGN="LEFT" WIDTH="100%" SIZE="2">
    <p><a NAME="Text"></a><a HREF="#Home"><font SIZE="4"><b><img SRC="back.gif" ALIGN="BOTTOM"
    BORDER="0" VSPACE="0" HSPACE="0" LOOP="0" width="12" height="12"></b></font></a></p>
    <p><font SIZE="4"><b>Molecular mechanisms involved in MHC-restricted antigen presentation
    and recognition by T cells</b></font></p>
    <p>Vaccination with DNA- or peptide-based vaccines relies on T cell recognition, either in
    the afferent (T cell-dependent antibody-mediated responses) or in both the afferent and
    effector limbs of the immune response (T cell-mediated responses - e.g., delayed
    hypersensitivity). In order to induce protective immunity, epitopes contained in these
    &quot;new generation&quot; subunit vaccines must i) match epitopes naturally presented to
    the immune system during infection; ii) be recognized by the entire target population, and
    iii) induce an immune response of an adequate isotype. The central event in the cellular
    immune response to invading microorganisms is the specific recognition of foreign peptides
    bound to major histocompatibility complex (MHC) molecules by the <font FACE="Symbol">a</font>ß
    T cell receptor (TCR). In most immunological systems, including post-infectious protective
    immune responses, only some peptides from a large number of potential candidates are
    actually the target of a vigorous immune response. The epitopes to which a natural immune
    response is generated upon immunization with the whole protein are called dominant
    epitopes. Immunization with overlapping peptides induces suboptimal responses to
    additional peptides, the so-called &quot;cryptic&quot; epitopes. A large number of factors
    influence dominance and crypticity of peptide epitopes, basically availability for MHC
    binding, MHC binding itself, and the recognition of the MHC:peptide complex by T cells via
    their antigen receptors (1). While peptide vaccines may bind directly to the MHC,
    recombinant vaccines must undergo proteolytic processing through the MHC class II pathway
    in endosomal vesicles, and expression products of DNA vaccines enter both the
    cytoplasmic/endoplasmic reticulum MHC class I pathway and can also be uptaken by
    professional antigen-presenting cells (2). The steps involved in antigen presentation and
    recognition by T cells that may influence the selection of antigenic protective epitopes
    from whole proteins will be reviewed.</p>
    <p><font SIZE="4"><b>Availability of peptides for MHC binding in antigen presentation
    pathways</b></font></p>
    <p>Several factors influence the availability of a given peptide sequence for processing
    and presentation, at both the quantitative and qualitative levels. An abundant protein has
    a higher chance of yielding peptides available for presentation. In intracellular
    eukaryotic parasites like <i>Trypanosoma cruzi</i>, proteins that are secreted may be a
    major source of peptides for the MHC class I presentation pathway (3). Cleavage
    site-specific proteases and transporter proteins involved in the processing of protein
    antigens into peptides also seem to play a role in the selection of antigenic peptides.
    Proteasomes, which generate peptides from cytoplasmic proteins for the class I pathway,
    seem to have preferred cleavage sites flanking dominant CD8+ T cell epitopes in protein
    sequences (4). The human TAP (transporter associated with antigen presentation) molecule
    selects peptides according to a binding motif on their three amino-terminal and two
    carboxy-terminal residues (5). Cathepsin and carboxypeptidase, which are present in the
    phagolysosomal compartment and are involved in proteolytic processing of endocytosed
    proteins in the MHC class II pathway, display preferential cleavage of dibasic (RR, KK, KR
    or RK) sites (6). Peptides eluted from human HLA class II molecules frequently display
    proline residues close to their N-terminus, consistent with N-aminopeptidase activity (7).</p>
    <p><font SIZE="4"><b>HLA polymorphism and binding of T cell epitopes</b></font></p>
    <p>T cells recognize antigen via the variable regions of the clonotypic T cell antigen
    receptor molecule (TCR). Zinkernagel's (8) pioneering experiments and succeeding
    refinements performed by many authors disclosed that T cells recognize antigen as
    proteolytic peptide fragments together with the individual's own MHC molecule. CD8+ T
    cells recognize peptide antigen as 8-10-residue-long peptides embedded into the
    antigen-binding groove of MHC class I molecules (the products of gene loci H2-K, -S, and
    -L in mice; HLA-A, -B and -C in humans). CD4+ T cells recognize peptide fragments of 13
    residues or longer embedded into the open-ended antigen-binding groove of MHC class II
    molecules (the products of gene loci H2 I-A and I-E in mice; HLA-DR, -DQ and -DP in
    humans).</p>
    <p>The structure of MHC molecules complexed with different peptides was solved by X-ray
    crystallography studies. Bound peptides are buried in the antigen-binding groove formed
    between the helices of the MHC molecule, leaving only a few of their side chains available
    for direct TCR contact (9-12). Peptides bind to MHC class II in an extended conformation
    where the side chain of each amino acid residue is rotated 120<sup>o</sup> relative to the
    previous one; nine residues lie within the antigen-binding groove (10) while flanking
    residues may also interact with the TCR (13). A network of hydrogen bonds between the
    peptide main chain and residues on the groove assures a tight interaction between peptide
    and MHC; specific binding of certain peptides to an MHC molecule comes from the
    interaction of peptide side chains with the irregular surface of the floor and sides of
    the groove, the &quot;pockets&quot; and ridges formed by the protrusion of MHC residues.
    The major pockets in the floor of the groove of HLA-DR molecules are occupied by the side
    chains of residues 1, 4, 6, and 9 of the bound peptide (10,11).</p>
    <p>Each human individual carries up to twelve different alleles from the six HLA class I
    and class II loci. HLA molecules are highly polymorphic, with more than 600 allelic forms
    for HLA class I and class II (14). This polymorphism is concentrated in the region
    encoding the peptide-binding groove, yielding very diverse amino acid sequences in this
    region among different HLA alleles. Thus, most pockets in the HLA groove are shaped by
    clusters of polymorphic HLA residues, with peculiar chemical and size characteristics in
    different HLA molecules. For example, a positively charged side chain (e.g. lysine) in one
    HLA molecule may interact preferentially with negatively charged residues (e.g. aspartic
    or glutamic acid) in a bound peptide; conversely, a hydrophobic residue in the same
    position of another HLA molecule may only accept peptides with hydrophobic residues in the
    corresponding position (15). It follows that each allelic HLA molecule only binds peptides
    with amino acid sequences that are capable of interacting with its antigen-binding groove.
    These preferences can be evident after the sequence alignment of peptides known to
    interact with a given HLA molecule, disclosing certain peptide positions where amino acids
    with similar side chains occur at increased frequency, the peptide-binding preferences or
    &quot;motifs&quot; (16). The study of large, MHC-selected peptide pools, and the
    sequencing of endogenous MHC-bound peptides (17) or the analysis of large peptide pools
    from phage display libraries selected by binding to HLA-DR (18) allowed a generalization
    of rules for peptide binding to individual HLA molecules (15,19). Single-chain
    substitution of &quot;designer&quot; (optimally binding) peptides led to the establishment
    of motifs based on quantitative matrices, where the effect of each amino acid residue in
    each peptide position was taken into account, generating a numeric score, rather than the
    all-or-none result for simple motifs (20). Further study of peptides eluted from HLA-DR
    and DQ suggested that residues flanking the bound nonamers may influence overall peptide
    binding preferences in an allele-specific manner (21), confirming what was observed in the
    mouse system (6). Since each of the different allelic HLA molecules has different
    peptide-binding preferences, a distinct set of epitopes from a given protein antigen will
    be presented to T cells in each individual bearing a different HLA molecule.</p>
    <p>On the whole, this implies that different individuals may be able to recognize
    different antigenic peptides from a given protein. This interindividual variation of
    antigen recognition can be a problem for selecting immunogens for DNA, recombinant and
    peptide vaccines, since they must contain epitopes recognized by individuals with a wide
    range of different HLA molecules. This can be circumvented by including additional
    epitopes that can be presented by different HLA alleles covering most of the population,
    which can increase the complexity of the vaccine. On the other hand, it has been observed
    that several distinct HLA class I molecules share peptide-binding preferences
    (&quot;supermotifs&quot;) and can bind certain common peptides with high affinity. Thus,
    the identification of single peptides that can bind multiple HLA types, the so-called
    &quot;promiscuous&quot; epitopes, could lead to effective coverage of the human population
    by a peptide-based vaccine (19).</p>
    <p><font SIZE="4"><b>T cell receptor recognition of MHC/peptide complex: structure and
    function</b></font></p>
    <p>Recently, the structure of two trimolecular complexes (TCR bound to MHC/peptide) was
    solved: a complex containing a human <font FACE="Symbol">a</font>ß TCR bound to an HLA-A2
    molecule loaded with a nonapeptide derived from the HTLV-1 virus (22) and a mouse TCR
    complexed to an H-2Kb/peptide ligand (23). In both structures, TCR V<font FACE="Symbol">a</font>Vß
    regions interact with the composite surface made of residues belonging to the antigenic
    peptide and to both MHC <font FACE="Symbol">a</font>1 and <font FACE="Symbol">a</font>2
    helices. The TCR is oriented diagonally across the peptide-binding site of HLA-A2, with
    the TCR V<font FACE="Symbol">a</font> contacting the left end of the peptide-binding
    groove, toward the amino-terminal part of the MHC <font FACE="Symbol">a</font>1 helix and
    of the peptide, while the TCR Vß is positioned toward the carboxy terminus of the
    peptide. The third complementarity-determining regions (CDR3) of the TCR V<font
    FACE="Symbol">a</font> and Vß chains, which display extremely diverse sequences due to
    junctional diversity that occurred during VDJ rearrangement, contact the center of the
    bound peptide and seem to have extensive conformational flexibility. In TCR:MHC class
    II:peptide complexes, a similar orientation seems to apply; however, N- and C-terminal
    residues flanking the peptide's nine MHC-bound residues may interact with the TCR (13).
    Flanking residues may also hinder TCR interaction with MHC/peptide complex (6,13).</p>
    <p>Along a peptide, the side chains of certain residues interact with the TCR (TCR contact
    residues) while residues in other positions interact with the MHC molecule (MHC contact
    residues or anchor residues). Substitutions at MHC contact residues may yield peptide
    analogs binding MHC with different avidities from the parent peptide. Avidity of the
    peptide for MHC and dosage seems to influence the cytokine profile resulting after
    immunization. It has been reported that among peptide analogs binding to MHC with
    different affinities over a 10,000-fold range, the higher affinity peptide always elicits
    a T1-type cytokine profile, while the lower affinity analog yields a T2-type profile at
    low doses (24) but a T1-type profile at higher doses (25). Peptide analogs with
    conservative substitutions at TCR contact residues may be full antagonists or partial
    agonists, inducing anergy or partial or differential T cell activation (26,27).</p>
    <p><font SIZE="4"><b>Prediction of T cell epitopes</b></font></p>
    <p>Ideally, modern subunit vaccines should contain highly immunogenic T cell epitopes
    intentionally chosen from the available pathogen proteins. However, a comprehensive
    screening of T cell epitopes by random cloning or direct synthesis of overlapping peptides
    from large numbers of gene sequences is often impossible. With the growing knowledge on
    the nature of TCR recognition of the MHC-peptide complex, several investigators tried to
    develop methods to predict T cell epitopes. The first algorithms, based on the simple
    alignment of antigenic peptides (and often disregarding diversity of MHC alleles), were no
    better than random when tested on larger databases (28). MHC allele-specific
    peptide-binding motifs were used in the prediction of T cell epitopes (29-31), minimizing
    the number of peptides to be synthesized and assayed. The success of MHC-based epitope
    prediction seems to lie in the filtering out of potential T cell epitopes, since most
    natural peptides seem to lack the capacity to interact with MHC molecules (20). MHC
    binding algorithms based on quantitative matrices yielded scores which correlated with
    binding affinity (20) like the TEPITOPE described for up to 25 distinct HLA-DR molecules
    (15), or the Internet server BIMAS (32) for several mouse and human MHC class I alleles,
    which permit the selection of high affinity binding peptides, the ones with the greater
    chance of eliciting T cell responses (33).</p>
    <p>From the vaccine immunologist's point of view, however, identification of MHC
    allele-specific T cell epitopes may not be enough, since one is searching for vaccine
    epitopes that can effectively cover the human population. This implies the identification
    of &quot;promiscuous&quot; epitopes that can bind to several MHC alleles whose combined
    frequency in the population approaches 100%. The use of HLA class I supermotifs (19), the
    alignment of peptides binding to several distinct HLA-DR molecules with TEPITOPE (15) and
    the identification of sequences containing clustered motifs for several distinct MHC
    molecules (34; Cunha-Neto E and Gruber A, unpublished observations) can identify such
    potentially &quot;promiscuous&quot; epitopes.</p>
    <p><font SIZE="4"><b>Escape from presentation and recognition by &quot;molecular
    evolution&quot;</b></font></p>
    <p>For millions of years pathogens have evolved molecular mechanisms to escape effective
    presentation and recognition to the immune system by variation of antigenic protein
    sequence, after interplaying with the host's immune system (35). Antigen processing and
    peptide transport mechanisms are ancient in evolutionary terms. The major features of the
    MHC peptide-binding groove, like the hydrophobic nature of most pockets, leading to
    preferential binding of peptides with hydrophobic residues, are phylogenetically conserved
    from cartilaginous fish and sharks to man (~250 million years) (36). Sequence variation in
    or flanking an epitope may lead to an inadequate response (37) by several mechanisms such
    as loss of processing signals (e.g., loss of sites for sequence-specific proteases or a
    different folding pattern), epitope competition and epitope sequestration (6,38),
    abrogation of MHC binding (39), antagonism or partial agonism of T cell receptor
    signalling (40), establishment of ineffective &quot;mock&quot; dominant epitopes (e.g.,
    those in variant regions of CS proteins from <i>Plasmodia</i>, stronger than those in
    conserved regions), and molecular mimicry with self or abundant environmental epitopes
    (41,42). In the case of molecular mimicry between host and pathogen epitopes, it is
    important to identify potentially pathogenic crossreactive epitopes to be excluded from a
    protective immunogen, as in the case of <i>Streptococcus pyogenes</i> M protein and
    rheumatic fever (43). Tandemly repeated domains of parasitic protozoan proteins, which are
    immunodominant B-cell epitopes, display nonrandom amino acid usage with few hydrophobic
    residues leading to deficient binding to MHC and T cell recognition (Abel LCJ, Gruber A
    and Cunha-Neto E, unpublished observations; 44). Thus, the immune response of infected
    individual to &quot;intact&quot; pathogen protein antigens may reflect the evolutionary
    success of the parasite. DNA or recombinant vaccines encoding whole proteins or their
    genes will carry most molecular sequence variation escape mechanisms evolved by the
    pathogen. Immunization with substituted synthetic peptides or DNA/proteins can present
    neoepitopes or alter the hierarchy of dominant/cryptic T cell epitopes, bypassing
    recognition escape mechanisms (45).</p>
    <p><font SIZE="4"><b>Concluding remarks</b></font></p>
    <p>The astonishing rate at which knowledge of T cell antigen recognition has been
    accumulating in the last decade is setting the stage for the comprehensive search for
    ideal epitopes and further &quot;engineering&quot; of immunogens from pathogens aiming at
    better vaccines. If widely applied, such knowledge will foster the production of several
    novel, highly effective subunit vaccines.</p>
    <hr ALIGN="LEFT" WIDTH="100%" SIZE="2">
    <p><a NAME="References"></a><font COLOR="#00007F" SIZE="4"><b>References</b></font> <a
    HREF="#Home"><font SIZE="4"><b><img SRC="back.gif" ALIGN="BOTTOM" BORDER="0" VSPACE="0"
    HSPACE="0" LOOP="0" width="12" height="12"></b></font></a></p>
    <p>1. Sercarz EE, Lehmann PV, Ametani A, Benichou G, Miller A &amp; Moudgil K (1993).
    Dominance and crypticity of T cell antigenic determinants. <i>Annual Review of Immunology</i>,
    11: 729-766.</p>
    <p>2. Fu TM, Ulmer JB, Caulfield MJ, Deck RR, Friedman A, Wang S &amp; Donnely JJ (1997).
    Priming of cytotoxic T lymphocytes by DNA vaccines: requirement for professional antigen
    presenting cells and evidence for antigen transfer from myocytes. <i>Molecular Medicine</i>,
    3: 362-371.</p>
    <p>3. Garg N, Nunes MP &amp; Tarleton RL (1997). Delivery by <i>Trypanosoma cruzi</i> of
    proteins into the MHC class I antigen. <i>Journal of Immunology</i>, 158: 3293-3302.<br>
    </p>
    <p>4. Niedermann G, Butz S, Ihlenfelt HG, Grimm R, Lucchiari M, Hoschutzky H, Jung G,
    Maier B &amp; Eichmann K (1995). Contribution of proteasome-mediated proteolysis to the
    hierarchy of epitopes presented by major histocompatibility complex class I molecules. <i>Immunity</i>,
    2: 289-299.</p>
    <p>5. Van Endert PM, Riganelli D, Greco G, Fleischhauer K, Sidney J, Sette A &amp; Bach JF
    (1995). The peptide binding motif for the human transporter associated with antigen
    processing. <i>Journal of Experimental Medicine</i>, 182: 1883-1895.<br>
    </p>
    <p>6. Moudgil KD, Deng H, Nanda NK, Grewal IS, Ametani A &amp; Sercarz EE (1996). Antigen
    processing and T cell repertoires as crucial aleatory features in induction of
    autoimmunity. <i>Journal of Autoimmunity</i>, 9: 227-234.<br>
    </p>
    <p>7. Falk K, Rotzschke O, Stevanovic S, Jung G &amp; Rammensee HG (1994). Pool sequencing
    of natural HLA-DR, DQ and DP ligands reveals detailed peptide motifs, constraints of
    processing, and general rules. <i>Immunogenetics</i>, 39: 230-242.<br>
    </p>
    <p>8. Zinkernagel RM &amp; Doherty P (1974). Restriction of <i>in vitro</i> T
    cell-mediated cytotoxicity in lymphocytic choriomeningitis within a syngeneic or
    semiallogeneic system. <i>Nature</i>, 248: 701-702.<br>
    </p>
    <p>9. Bjorkman PJ, Saper MA, Samraoui B, Bennett WS, Strominger JL &amp; Wiley DC (1987).
    The foreign antigen binding site and T cell recognition regions of class I
    histocompatibility antigens. <i>Nature</i>, 329: 506-510.<br>
    </p>
    <p>10. Stern LJ, Brown JH, Jardetzky TS, Gorga JC, Urban RG, Strominger JL &amp; Wiley DC
    (1994). Crystal structure of the human class II MHC protein MHC-DR1 complexed with an
    influenza virus peptide. <i>Nature</i>, 368: 215-221.<br>
    </p>
    <p>11. Dessen A, Martin Lawrence C, Cupo S, Zaller DM &amp; Wiley DM (1997). X-ray crystal
    structure of HLA-DR4 (DRA*0101, DRB1*0401) complexed with a peptide from human collagen
    II. <i>Immunity</i>, 7: 473-481.</p>
    <p>12. Engelhard VH (1994). Structure of peptides associated with class I and class II MHC
    molecules. <i>Annual Review of Immunology</i>, 12: 181-207.</p>
    <p>13. Carson RT, Vignali KM, Woodland DL &amp; Vignali DAA (1997). T cell receptor
    recognition of MHC class II-bound peptide flanking residues enhances immunogenicity and
    results in altered TCR V region usage. <i>Immunity</i>, 7: 387-399.</p>
    <p>14. Charron D (Editor) (1997).<i> HLA: Genetic Diversity, Functional and Medical
    Implications</i>. Proceedings of the XII International Workshop on HLA. EDK Medical
    Publishers, Sèvres, France.</p>
    <p>15. Hammer J, Sturniolo T &amp; Sinigaglia F (1997). HLA class II binding specificity
    and autoimmunity. <i>Advances in Immunology</i>, 66: 67-100.<br>
    </p>
    <p>16. Rammensee HG (1995). MHC ligands and peptide motifs: first listing. <i>Immunogenetics</i>,
    41: 178-228 (Internet site with compiled MHC binding peptides and motifs:
    http://wehih.wehi.edu.au/mhcpep/).<br>
    </p>
    <p>17. Rötzschke O &amp; Falk K (1994). Origin, structure and motifs of naturally
    processed MHC class II ligands. <i>Current Opinion in Immunology</i>, 6: 45-51.</p>
    <p>18. Hammer J (1995). New methods to predict MHC-binding sequences within protein
    antigens. <i>Current Opinion in Immunology</i>, 7: 263-269.</p>
    <p>19. Sidney J, Grey HM, Ralph RT &amp; Sette A (1996). Practical, biochemical and
    evolutionary implications of the discovery of HLA class I supermotifs. <i>Immunology</i> <i>Today</i>,
    17: 262-266.</p>
    <p>20. Hammer J, Bono E, Gallazzi F, Belunis C, Nagy Z &amp; Sinigaglia F (1994). Precise
    prediction of major histocompatibility complex class II molecules by anchor combinations. <i>Proceedings
    of the National Academy of Sciences, USA</i>, 91: 4456-4460.</p>
    <p>21. Godkin AJ, Davenport MP, Willis A, Jewell DP &amp; Hill AV (1998). Use of complete
    eluted peptide sequence data from HLA-DR and -DQ molecules to predict T cell epitopes, and
    the influence of the nonbinding terminal regions of ligands in epitope selection. <i>Journal
    of Immunology</i>, 161: 850-858.<br>
    </p>
    <p>22. Garboczi DN, Ghosh P, Utz U, Fan QR, Biddison WE &amp; Wiley DC (1996). Structure
    of the complex between human T-cell receptor, viral peptide and HLA-A2. <i>Nature</i>,
    384: 134-141.<br>
    </p>
    <p>23. Garcia KC, Degano M, Stanfield RL, Brunmark A, Jackson MR, Peterson PA, Teyton L
    &amp; Wilson IA (1996). An alphabeta T cell receptor structure at 2.5 A and its
    orientation in the TCR-MHC complex. <i>Science</i>, 274: 209-219.<br>
    </p>
    <p>24. Kumar V, Bhardwaj V, Soares L, Alexander J, Sette A &amp; Sercarz EE (1995). MHC
    binding affinity of an antigenic determinant is crucial for the differential secretion of
    interleukin 4/5 or interferon gamma by T cells. <i>Proceedings of the National Academy of
    Sciences, USA</i>, 92: 9510-9514.</p>
    <p>25. Chaturvedi P, Yu Q, Southwood S, Sette A &amp; Singh B (1996). Peptide analogs with
    different affinities for MHC alter the cytokine profile of helper T cells. <i>International
    Immunology</i>, 8: 745-755.</p>
    <p>26. Windhagen A, Scholz C, Hollsberg P, Fukaura H, Sette A &amp; Hafler DA (1995).
    Modulation of cytokine patterns of human autoreactive T cell clones by a single amino acid
    substitution of their peptide ligand. <i>Immunity</i>, 2: 373-380.</p>
    <p>27. Kersh GJ &amp; Allen PM (1996). Essential flexibility in the T-cell recognition of
    antigen. <i>Nature</i>, 380: 495-500.<br>
    </p>
    <p>28. Deavin AJ, Auton TR &amp; Greaney PJ (1996). Statistical comparison of established
    T-cell epitope predictors against a large database of human and murine antigens. <i>Molecular
    Immunology</i>, 33: 145-155.<br>
    </p>
    <p>29. Rotzschke O, Falk K, Stevanovic S, Jung G, Walden P &amp; Rammensee HG (1991).
    Exact prediction of a natural T cell epitope. <i>European Journal of Immunology</i>, 21:
    2891-2894.<br>
    </p>
    <p>30. Stevanovic S &amp; Rammensee HG (1994). Identification of T-cell epitopes using
    allele-specific ligand motifs. <i>Behring Institute Mitteilungen</i>, 95: 7-13.</p>
    <p>31. Celis E, Fikes J, Wentworth P, Sidney J, Southwood S, Maewal A, Del Guercio MF,
    Sette A &amp; Livingston B (1994). Identification of potential CTL epitopes of
    tumor-associated antigen MAGE-1 for five common HLA-A alleles. <i>Molecular Immunology</i>,
    31: 1423-1430.<br>
    </p>
    <p>32. Parker KC, Bednarek MA &amp; Coligan JE (1994). Scheme for ranking potential HLA-A2
    binding peptides based on independent binding of individual peptide side-chains. <i>Journal
    of Immunology</i>, 152: 163-175 (Internet site for prediction of binding to several
    different MHC class I molecules: http://bimas.dcrt.nih.gov/molbio/hla_bind/).<br>
    </p>
    <p>33. Sette A, Vittiello A, Reherman B, Fowler P, Nayersina R, Kast WM, Melief CJ, Oeroff
    C, Yuan L, Ruppert L &amp; Grey H (1994). The relationship between class I binding
    affinity and immunogenicity of potential cytotoxic T cell epitopes. <i>Journal of
    Immunology</i>, 153: 5586-5592.<br>
    </p>
    <p>34. Meister GE, Roberts CGP, Berzofsky J &amp; De Groot A (1995). Two novel T cell
    epitope prediction algorithms based on MHC-binding motifs; comparison of predicted and
    published epitopes from <i>Mycobacterium tuberculosis</i> and HIV protein sequences. <i>Vaccine</i>,
    13: 581-591.<br>
    </p>
    <p>35. Nagy ZA &amp; Adorini L (1989). Why peptides? Their possible role in the evolution
    of MHC-restricted T-cell recognition. <i>Immunology Today</i>, 10: 132-138.</p>
    <p>36. Kasahara M, McKinney EC, Flajnik MF &amp; Ishibashi T (1993). The evolutionary
    origin of the major histocompatibility complex: polymorphism of class II alpha chain genes
    in the cartilaginous fish. <i>European Journal of Immunology</i>, 23: 2160-2165.<br>
    </p>
    <p>37. Johnson RP, Trocha A, Buchanan TM &amp; Walker BD (1992). Identification of
    overlapping HLA class I-restricted cytotoxic T cell epitopes in a conserved region of the
    human immunodeficiency virus type 1 envelope glycoprotein: definition of minimum epitopes
    and analysis of effects of sequence variation. <i>Journal of Experimental Medicine</i>,
    175: 961-971.<br>
    </p>
    <p>38. Sercarz E, Gammon G &amp; Palmer M (1990). T cell dominance and the vaccine
    problem: modifying effects on immunogenicity by residues at a distance from the site of T
    cell recognition. <i>Seminars in Immunology</i>, 2: 297-305.</p>
    <p>39. Wrightham MN, Stewart JP, Janjua NJ, Pepper SD, Sample C, Rooney CM &amp; Arrand JR
    (1995). Antigenic and sequence variation in the C-terminal unique domain of the
    Epstein-Barr virus nuclear antigen EBNA-1. <i>Virology</i>, 208: 521-530.<br>
    </p>
    <p>40. Franco A, Ferrari C, Sette A &amp; Chisari FV (1995). Viral mutations, TCR
    antagonism and escape from the immune response. <i>Current Opinion in Immunology</i>, 7:
    524-531.</p>
    <p>41. Good MF, Zevering Y, Currier J &amp; Bilsbough J (1993). &quot;Original antigenic
    sin&quot;, T cell memory, and malaria sporozoite immunity: a hypothesis for immune
    evasion. <i>Parasite Immunology</i>, 15: 187-193.<br>
    </p>
    <p>42. Loftus DJ, Castelli C, Clay TM, Squarcina P, Marincola FM, Nishimura MI, Parmiani
    G, Appella E &amp; Rivoltini L (1996). Identification of epitope mimics recognized by CTL
    reactive to the melanoma/melanocyte-derived peptide MART-1. <i>Journal of Experimental
    Medicine</i>, 184: 647-657.<br>
    </p>
    <p>43. Guilherme L, Cunha-Neto E, Coelho V, Snitcowsky R, Pomerantzeff PMA, Assis RV,
    Pedra F, Neumann J, Goldberg A, Patarroyo ME, Pileggi F &amp; Kalil J (1995). Human
    infiltrating T cell clones from rheumatic heart disease recognize both streptococcal and
    cardiac proteins. <i>Circulation</i>, 92: 415-420.<br>
    </p>
    <p>44. Calvo-Calle JM, Hammer J, Sinigaglia F, Clavijo P, Moya-Castro ZR &amp; Nardin EH
    (1997). Binding of malaria T cell epitopes to DR and DQ molecules <i>in vitro</i>
    correlates with immunogenicity <i>in vivo</i>: identification of a universal T cell
    epitope in the <i>Plasmodium falciparum</i> circumsporozoite protein. <i>Journal of
    Immunology</i>, 159: 1362-1373.<br>
    </p>
    <p>45. Fu TM, Friedman A, Ulmer JB, Liu MA &amp; Donnelly JJ (1997). Protective cellular
    immunity: cytotoxic T-lymphocyte responses against dominant and recessive epitopes of
    influenza virus nucleoprotein induced by DNA immunization. <i>Journal of Virology</i>, 71:
    2715-2721.<br>
    </p>
    <p><a NAME="Correspondence"></a><font SIZE="4"><font COLOR="#00007F"><b>Correspondence and
    Footnotes</b></font><font COLOR="#000000"> </font><a HREF="#Home"><b><img SRC="back.gif"
    ALIGN="BOTTOM" BORDER="0" VSPACE="0" HSPACE="0" LOOP="0" width="12" height="12"></b></a></font></p>
    <p><b>Address for correspondence: </b>E. Cunha-Neto, Laboratório de Imunologia de
    Transplantes, Instituto do Coração, Av. Enéas C. Aguiar, 500, 3º andar, 05403-000 São
    Paulo, SP, Brasil. Fax: +55-11-282-9350. E-mail: <a HREF="mailto:edecunha@usp.br">edecunha@usp.br</a></p>
    <p>Presented at the International Symposium &quot;The Third Revolution on Vaccines: DNA
    Vaccines&quot;, Belo Horizonte, MG, Brasil, November 3-7, 1997. Research supported by CNPq
    and FAPESP. Received November 6, 1998. Accepted November 27, 1998.</td>
    <td WIDTH="5%"></td>
  </tr>
</table>

<p align="center">&nbsp;</p>

<p align="center"><img src="http:/fbpe/img/revistas/bjmbr/barrinha.jpg" width="599"
height="10"><br>
</p>

<p ALIGN="center"><font FACE="Symbol">Ó</font><font COLOR="#000080"><i><b>1997, 1998
Brazilian Journal of Medical and Biological Research</b></i></font>.<br>
<font SIZE="2" COLOR="#000080"><b>Av. Bandeirantes, 3900<br>
14049-900 Ribeirão Preto, SP - Brasil<br>
Telephone and Fax: (55 16) 633-3825<br>
</b></font><br>
<br>
</p>

<p>&nbsp;</p>
</body>
</html>
